be/src/vec/sink/vtablet_sink.cpp

```diff
@@ -1014,6 +1043,31 @@ Status VOlapTableSink::open(RuntimeState* state) {
     return Status::OK();
 }
 
+void VOlapTableSink::_open_partition(const VOlapTablePartition* partition, uint32_t tablet_index) {
```

Comment:
we don't need `tablet_index` passed to this function.
Tablets shoud be deduced using `partition` only.